home/audio.asm

```diff
@@ -252,18 +273,8 @@ WaitSFX::
 .wait
 	call DelayFrame
 .handleLoop
-	ld hl, wChannel5Flags
-	bit 0, [hl]
-	jr nz, .wait
-	ld hl, wChannel6Flags
-	bit 0, [hl]
-	jr nz, .wait
-	ld hl, wChannel7Flags
-	bit 0, [hl]
-	jr nz, .wait
-	ld hl, wChannel8Flags
-	bit 0, [hl]
-	jr nz, .wait
+	call CheckSFX
+	jr c, .wait
```

Comment:
Looks like the return code of `CheckSFX` can just be simplified to `z/nz` and have `bit SOUND_CHANNEL_ON, a / ret`.

Reply:
The call sites already expect c but there aren't many, i'll change it